src/pretix/base/services/mail.py

```diff
@@ -332,3 +340,75 @@ def render_mail(template, context):
         tpl = get_template(template)
         body = tpl.render(context)
     return body
+
+
+def replace_images_with_cid_paths(body_html):
+    if body_html:
+        email = BeautifulSoup(body_html, "lxml")
+        image_counter = 1
+        cid_images = []
+        for image in email.findAll('img'):
+            cid_id = "image_%s" % (image_counter)
+            image_counter = image_counter + 1
+            original_image_src = image['src']
+
+            image['src'] = "cid:%s" % (cid_id)
+
+            cid_images.append({
+                'src': original_image_src,
+                'cid_id': cid_id
+            })
+
+        return email.prettify(), cid_images
+    else:
+        return body_html, []
+
+
+def attach_cid_images(msg, cid_images, verify_ssl=True):
+    if cid_images and len(cid_images) > 0:
+
+        msg.mixed_subtype = 'related'
+        for image in cid_images:
+            try:
+                mime_image = convert_image_to_cid(
+                    image['src'], image['cid_id'], verify_ssl)
+                if mime_image:
+                    msg.attach(mime_image)
+            except Exception as e:
+                print("ERROR attaching CID image %s[%s] %s" % (image['cid_id'], image['src'], str(e)))
```

Comment:
Could you replace this by

```
except:
    logger.exception("ERROR attaching CID image %s[%s]" % (image['cid_id'], image['src']))
```

This will automatically log it to the right place and also add a stacktrace automatically